src/motion_click_app.c

```diff
@@ -56,21 +56,15 @@ static void print_usage(const char *program)
         program);
 }
 
-static int parse_cmd_opts(int argc, char *argv[], uint8_t *gpio)
+static int8_t parse_cmd_opts(int argc, char *argv[], uint8_t *gpio)
```

Comment:
As I know the return value range here, that's why I took int8_t. But your point makes much sense so I will reverse it.